src/mkdocs_gallery/scrapers.py

```diff
@@ -310,7 +310,7 @@ def mayavi_scraper(block, script: GalleryScript):
     e = mlab.get_engine()
     for scene, image_path in zip(e.scenes, image_path_iterator):
         try:
-            mlab.savefig(image_path, figure=scene)
+            mlab.savefig(str(image_path), figure=scene)
```

Comment:
This gave me an error when a PosixPath was passed in, so I think it's safest to convert to string here.